src/Cabrio.java

```diff
@@ -2,16 +2,14 @@
     public static String model;
```

Comment:
Cabrio is a class. Regular class, because we can create multiple objects of cabrio cars. Your class is not regular. It has static fields, why does it have static field if it doesn't make sense to do so?